src/style.css

```
@@ -1 +1,82 @@
-/* Styles go here */
+body {
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: stretch;
+  margin: auto;
+  padding: 0 50px;
+  height: 100%;
+}
+
+.moyo_logo {
+  width: 40px;
+  height: 40px;
+  margin: 10px 0;
+}
+
+.navigation {
+  display: flex;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  align-items: stretch;
+}
+
+.nav__list {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
```

You already applied space-between in `header`, so it's not needed here:
```suggestion
.nav__list {
  padding: 0;
  margin: 0;
  list-style: none;
  display: flex;
}
```